pilot/pkg/status/ledger.go

```diff
@@ -28,7 +28,7 @@ func tryLedgerPut(configLedger ledger.Ledger, obj config.Config) {
 
 func tryLedgerDelete(configLedger ledger.Ledger, obj config.Config) {
 	key := config.Key(obj.GroupVersionKind.Kind, obj.Name, obj.Namespace)
-	if err := configLedger.Delete(key); err != nil {
+	if _, err := configLedger.Delete(key); err != nil {
```

Comment:
The `Delete` returns a `result` as well an error. Should we do anything with the result? It's a string.